src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import React from "react";
```